docs/content/entity-templates.md

```diff
@@ -53,6 +53,56 @@ public static class CreatureTemplate
 
 You can use this entity template to spawn a Creature entity. This can be done using the [MonoBehaviour](gameobject/world-commands.md) or [ECS](ecs/world-commands.md) workflow.
 
+### Feature Module Components
+
+To take advantage of some feature modules, you need to add the components that the modules will make use of.  This is done by calling the appropriate helper method for each desired feature module.
```

Comment:
Nit: double " " between the two sentences. Also, maybe change "add the components" to "add SpatialOS components".